README.md

```diff
@@ -46,15 +46,15 @@ USAGE:
     arbiter [FLAGS] [OPTIONS]
 
 FLAGS:
-        --api_key    Provide an Etherscan API key if function calls require it (e.g., providing a token that is not in
-                     the database).
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+        --api_key     Provide an Etherscan API key if function calls require it (e.g., providing a token that is not in
+                      the database).
+    -h, --help        Prints help information
+    -V, --version     Prints version information
 
 OPTIONS:
         --fee <fee>          Specifies the basis points for the pool. [1, 5, 30, 100] [default: 5]
         --token0 <token0>    Specifies the first token for a token pair. [default: ETH]
-        --token1 <token1>    Specifies the second token for a token pair which will be the numeraire. [default: USDC]
+        --token1 <token1>    Specifies the second token for a token pair which we be the numeraire. [default: USDC]
```

Comment:
typo?

Reply:
oops yeah, let me fix alongside some fmt stuff to make it pass the ci